### PR TITLE
[🐴] Appeal form for disabled DMs

### PR DESCRIPTION
**Pull request by mozzius**
mostly a straight copy of `LabelsOnMeDialog`, although I improved the onSubmit action by using `useMutation` and backported it to `LabelsOnMeDialog`

### CODE REVIEW

File: src/components/dms/MessagesListHeader.tsx [hunk 1]
```diff
@@ -176,6 +177,13 @@ function HeaderReady({
               ]}
               numberOfLines={1}>
               @{profile.handle}
+              {convoState.convo?.muted && (
+                <>
+                  {' '}
+                  &middot;{' '}
+                  <BellStroke size="xs" style={t.atoms.text_contrast_medium} />
+                </>
+              )}
```
Comment (estrattonbailey) [inline src/components/dms/MessagesListHeader.tsx]: Heh nice

File: src/screens/Messages/Conversation/ChatDisabled.tsx [hunk 2]
```diff
@@ -20,7 +30,136 @@ export function ChatDisabled() {
...
+            ...web({outline: 0}),
+            textDecorationLine: 'underline',
+            textDecorationColor: t.palette.primary_500,
+          },
+        ]}
+        onPress={control.open}
+        onPressIn={onPressIn}
+        onPressOut={onPressOut}>
+        <Trans>Appeal this decision</Trans>
+      </Text>
```
Comment (estrattonbailey) [inline src/screens/Messages/Conversation/ChatDisabled.tsx:67]: Rn this isn't a tabbable element, so not good for a11y. This is the hacky way we have atm to add an inline link style button: https://github.com/bluesky-social/social-app/blob/main/src/components/dms/MessageItem.tsx#L222

Comment (estrattonbailey) [reply src/screens/Messages/Conversation/ChatDisabled.tsx:67]: Basically it's a link with an `onPress` that disables the link handling

Comment (estrattonbailey) [reply src/screens/Messages/Conversation/ChatDisabled.tsx:67]: Could alternatively just use a `small` sized `Button`

**Review by estrattonbailey — COMMENTED** [review]
Looks great! Let's just do something more accessible for the trigger

File: src/screens/Messages/Conversation/ChatDisabled.tsx (hunk 2 above)
Comment (mozzius) [reply src/screens/Messages/Conversation/ChatDisabled.tsx:67]: I think a small button is the way to go here

**Review by estrattonbailey — APPROVED** [review]

**Pull request merged**